packages/graphql/tests/integration/pagination.int.test.ts

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
These strike me more as unit tests which should be living at the same level as the source code, i.e. `pagination.test.ts` in the same directory as `pagination.ts`.